doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -974,32 +974,42 @@ Generation of induction principles with ``Scheme``
    .. insertprodn scheme_kind sort_family
 
    .. prodn::
-      scheme_kind ::= Equality for @reference
-      | {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
-      sort_family ::= Set
-      | Prop
+      scheme_kind ::= {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
```

Comment:
I think you need to run doc_gram_rsts again.  If you want to have `Induction` et al in-line like this, add this line to `common.edit_mlg` just above `] (* end SPLICE *)`: `| schema_type`.  If you want to keep it as a separate nonterminal in the doc, edit `orderedGrammar`  so that `schema_type` appears just after `scheme_kind`--in which case you can use it as the title in your table.  Nice table!